lib/msal-core/src/telemetry/TelemetryUtils.ts

```diff
@@ -17,7 +17,7 @@ export const scrubTenantFromUri = (uri: string): String => {
     const pathParams = url.PathSegments;
 
     if (pathParams && pathParams.length >= 2) {
-        const tenantPosition = pathParams[1] ===  B2cAuthority.B2C_PREFIX ? 2 : 1;
+        const tenantPosition = pathParams[1] ===  "tfp" ? 2 : 1;
```

Comment:
Are we still using tfp for anything?

Reply:
Not sure, have we stopped doing this? We do use it for taking PII out of urls (tenants)

Reply:
My understanding was that all the b2c urls moved to something.b2clogin.com/something.onmicrosoft.com/policy and something.b2clogin.com/tenant_id/policy
@sameerag Was that your understanding as well?

Reply:
yes. The more I think, the more I just want to have a type and be done with it. :-). Most MSALs still have this logic + indication from the client that this is 'B2C'

Reply:
So does the distinction between AAD and B2C matter in this example? I think the tenant info will always be in position 1 now right?

Reply:
But, we need it for legacy apps compatibility, the migration may not be yet complete.

Reply:
@tnorling  can you share where we landed on this?

Reply:
@DarylThayil There are still cases where b2c authority urls will contain "tfp", so while this check was no longer a good check for the AuthorityFactory it is still needed for this use case here. This can remain as written.

Reply:
@tnorling @DarylThayil Which version should be used here? `B2cAuthority.B2C_PREFIX` or `tfp`?

Reply:
I'd probably just go with the ```tfp``` since this will likely be the only place we reference it. But functionally either works.

Reply:
Keeping this for now